sitemap.functions.php

```diff
@@ -365,6 +367,11 @@ function scan_url($url)
         return $depth--;
     }
 
+    if ($noindex && preg_match('/content="noindex"/', $html)) {
```

Comment:
This is not a reliable way to check for html. 
**Possible false positive**: The content may have been included as an attribute of some random tag. We're just looking for meta tags. It's also not applicable to _all_ meta tags.

**Possible false negative**: There is a lot of variety allowed by html. Here are some examples:

```
// Single quotes
content='noindex'
// Spaces around equality
content = "noindex"
// capitals
CONTENT="noindex"
```

We do need to account for those.

I'm okay merging this despite the false negatives, but false positives should be fixed before I can do so.

Reply:
if ($noindex && preg_match('/\<\s*meta\s+name\s*\=\s*\'*\"*\s*robots*\s*\'*\"*\s*content\s*\=\s*\'*\"*\s*noindex*\s*\'*\"*\s*\>/i', $html)) {